### PR TITLE
Add option to pass viewport sizing to PhantomJS

### PR DESCRIPTION
**Pull request by m90**
Would solve #97 

I made one mistake when doing the changes for that in [phantomic](https://github.com/mantoni/phantomic/pull/16) though, so we might want to fix this first? Will comment inline.

### CODE REVIEW

File: lib/phantom.js [hunk 1]
```diff
@@ -48,12 +48,14 @@ module.exports = function (b, opts) {
...
-      phantomjs      : phantomPath,
-      'web-security' : opts['web-security'],
-      'ignore-ssl-errors': opts['ignore-ssl-errors']
+      debug              : opts.debug,
+      port               : opts.port,
+      brout              : true,
+      phantomjs          : phantomPath,
+      'web-security'     : opts['web-security'],
+      'ignore-ssl-errors': opts['ignore-ssl-errors'],
+      viewportWidth      : opts['viewport-width'],
```
Comment (m90) [inline lib/phantom.js]: I did not respect the correct kebap-case everywhere when adding this to phantomic. We might want to fix that in there first, so we can get rid of that oddity in here.

Comment (m90) [inline lib/phantom.js]: PR for that would be here: https://github.com/mantoni/phantomic/pull/17

Comment (mantoni) [inline lib/phantom.js]: After changing this in phantomic, it needs to be updated here, right?

**Comment by mantoni** [comment]
Released in `v3.2.0` 🎉 

I just noticed that we both forgot about the `help.txt` file which should list the new options also ... it's getting a bit longish now ... 🤔 

Another thing that came to mind, maybe for later: It would be cool to default the viewport values with the current actual console size – should we detect that we're running in a tty.

**Comment by m90** [comment]
True that, I guess completeness trumps brevity so I will fix the help.txt real quick.

About the tty: cool idea, might be well worth opening an issue.

**Pull request merged**